### PR TITLE
Release 3.2.4

### PR DESCRIPTION
**Pull request by Goose-Of-War**
Newsletter crossword and compression

### CODE REVIEW

**Comment by Goose-Of-War** [comment]
#150 Yay 🥳🥳

File: templates/newsletters/2023-05-1/2023-05-1.njk
Comment (nishkalprakash) [inline templates/newsletters/2023-05-1/2023-05-1.njk:1]: The crossword scroll isnt loading

Comment (nishkalprakash) [reply templates/newsletters/2023-05-1/2023-05-1.njk:1]: image

Comment (nishkalprakash) [reply templates/newsletters/2023-05-1/2023-05-1.njk:1]: checked in mobile screen as well

**Review by nishkalprakash — CHANGES_REQUESTED** [review]

File: templates/newsletters/2023-05-1/2023-05-1.njk
Comment (Goose-Of-War) [reply templates/newsletters/2023-05-1/2023-05-1.njk:1]: Workin on it

**Review by nishkalprakash — APPROVED** [review]

**Review by nishkalprakash — COMMENTED** [review]
good work

**Pull request merged**